packages/server/src/migrations/data/data-version-manifest.json

```diff
@@ -73,5 +73,8 @@
   },
   "v21": {
     "serverVersion": "4.3.9"
+  },
+  "v22": {
+    "serverVersion": "4.3.11"
```

Comment:
Can we add a second post-deploy to reindex `ServiceRequest`?

Reply:
Sure thing!  I can also have it reindex `Task`, since that was previously updated to be in the Patient compartment